study-builder/studies/pancan/i18n/en.conf

```diff
@@ -17,21 +17,179 @@
 "prequal_sections_01_question_01_picklistOptions_02_optionLabelTemplate"="My child has been diagnosed with cancer and signing up with them or for them"
 "prequal_sections_01_question_01_validations_hintTemplate"="Please choose one of the above options."
 
-"prequal_diagnosed_primary_cancers_prompt"="What primary cancer(s) have you/your child been diagnosed with?"
-"prequal_sections_02_question_01_validations_01_hintTemplate_01"="If you/your child have been diagnosed with more than one cancer type, please select all that apply."
+"prequal_diagnosed_primary_cancer_list_prompt"="What primary cancer(s) have you/your child been diagnosed with? \n If you/your child have been diagnosed with more than one cancer type, please use the button below to add an additional diagnosis."
```

Comment:
The prompt is going into HTML, so I don't think `\n` is what you want. Maybe try `<br/>`. (Have we considered a tooltip?)

Reply:
HTML elements are not permitted  in the question prompts on the mobile apps if the new line is important for the formatting of the question.

Reply:
HTML elements in question prompt will get stripped.

Reply:
I would have recommended "additional info header" but I don't think web client supports that at the moment.

Reply:
Additional info header appears above the prompt in mobile.

Reply:
> HTML elements in question prompt will get stripped for clients that wants it trimmed down.

Yes, but that doesn't address my point. If that newline is important to the presentation, switching to a `</br>` will result in everything being squashed onto a single wrapped line.

Reply:
My guess is that it's shown on one line or two lines is not that important.

Reply:
What about `picklistLabel`? In the mobile apps, this is currently presented as a label placed below the prompt, using a lighter grey coloration & slight smaller font. Given the text here is ancillary information, that may be a good place for it.

Reply:
What about `picklistLabel`? In the mobile apps, this is currently presented as a label placed below the prompt, using a lighter grey coloration & slight smaller font. Given the text here is ancillary information, that may be a good place for it.

```json
{
  "prompt": "What primary cancer(s) have you/your child been diagnosed with?",
  "picklistLabel": "If you/your child have been diagnosed with more than one cancer type, please use the button below to add an additional diagnosis."
}
```

One other thing, is the you/your child wording correct? It reads like it should be "you have/your child has"- "your child have" is odd.

Reply:
separated into two text content blocks.
Text will be either: you OR your child depending on who.